mls-rs-provider-sqlite/src/lib.rs

```diff
@@ -170,4 +216,29 @@ mod tests {
 
         assert_eq!(current_schema, 1);
     }
+
+    #[test]
+    pub fn journal_mode_test() {
+        let db_path = Path::new("./test_db.sqlite");
```

Comment:
May you use [`tempdir`](https://docs.rs/tempfile/latest/tempfile/fn.tempdir.html)? That is the usual way to create files during tests and it takes care of cleaning automatically.